docs/api/classifier/confusion_matrix.rst

```diff
@@ -3,22 +3,22 @@
 Confusion Matrix
 ================
 
-The ``ConfusionMatrix`` visualizer is a ScoreVisualizer that takes a
-fitted scikit-learn classifier and a set of test X and y values and
+The ``ConfusionMatrix`` visualizer is a ``ScoreVisualizer`` that takes a
+fitted scikit-learn classifier and a set of test ``X`` and ``y`` values and
 returns a report showing how each of the test values predicted classes
 compare to their actual classes. Data scientists use confusion matrices
 to understand which classes are most easily confused. These provide
-similar information as what is available in a ClassificationReport, but
+similar information as what is available in a ``ClassificationReport``, but
 rather than top-level scores, they provide deeper insight into the
 classification of individual data points.
 
-Below are a few examples of using the ConfusionMatrix visualizer; more
+Below are a few examples of using the ``ConfusionMatrix`` visualizer; more
 information can be found by looking at the
 scikit-learn documentation on `confusion matrices <http://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html>`_.
 
 .. code:: python
 
-    from sklearn.datasets import load_digits
+    from sklearn.datasets import load_digits, load_iris
```

Comment:
Added `load_iris` to the import statement since it is called in the second code example.